docs/visualization/rgb.rst

```diff
@@ -4,6 +4,24 @@
 Creating color RGB images
 *************************
 
+RGB images can be produced using matplotlib's ability to make three-color
+images.  In general, an RGB image is an MxNx3 array, where M is the
+y-dimension, N is the x-dimension, and the length-3 layer represents red,
+green, and blue, respectively.  A fourth layer representing the alpha (opacity)
+value can be specified.
+
+Matplotlib has several tools for manipulating these colors at
+https://matplotlib.org/api/colors_api.html.
```

Comment:
could you use intersphinx to link to this page rather than the hardwired link? It should either work with `matplotlib.colors` (though that's not exactly for for the top of that page)

Reply:
How do you intersphinx link to a documentation page?  The page linked is not an apidoc, it's a .rst doc, despite the `api` in the url

Reply:
most of the docs pages are also in the intersphinx inventory, this one has the line:
  `matplotlib.colors -> 'api/colors_api.html#module-matplotlib.colors'`, so I would try `` `matplotlib.colors` ``